app/src/main/java/org/thoughtcrime/securesms/onboarding/LandingActivity.kt

```diff
@@ -117,7 +117,7 @@ class LandingActivity: BaseActionBarActivity() {
         }
 
         Column {
-            Column(modifier = Modifier
+            /*Column(modifier = Modifier
```

Comment:
Do we need this commented out code?

Reply:
fixed. just accidental commit during button text style bug hunting.